src/main/java/net/sf/jabref/importer/fetcher/IEEEXploreFetcher.java

```diff
@@ -172,16 +180,10 @@ public boolean processQuery(String query, ImportInspector dialog, OutputPrinter
 
         } catch (MalformedURLException e) {
             LOGGER.warn("Bad URL", e);
-        } catch (ConnectException e) {
+        } catch (ConnectException | UnknownHostException e) {
             status.showMessage(Localization.lang("Connection to IEEEXplore failed"), DIALOG_TITLE,
```

Comment:
Changed it to (already existing) "Could not connect to %0". Waiting with removal until the automatic one is done. ;-)